google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/v1p1beta1/NotificationConfigSnippets.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.examples.securitycenter.snippets.v1p1beta1;
+
+import com.google.cloud.securitycenter.v1p1beta1.CreateNotificationConfigRequest;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig.EventType;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig.StreamingConfig;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfigName;
+import com.google.cloud.securitycenter.v1p1beta1.OrganizationName;
+import com.google.cloud.securitycenter.v1p1beta1.SecurityCenterClient;
+import com.google.cloud.securitycenter.v1p1beta1.SecurityCenterClient.ListNotificationConfigsPagedResponse;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+/** Snippets for how to work with Notifications in Cloud SCC. */
+final class NotificationConfigSnippets {
+  private NotificationConfigSnippets() {}
+
+  public static NotificationConfig createNotificationConfig(
+      String organizationId, String pubsubTopic, String notificationConfigId) {
+
+    // [START scc_create_notification_config]
+    // TODO(UpdateMe) String organizationId = "{your-org-id}";
+    // TODO(UpdateMe) String notificationConfigId = "config-id";
+    // TODO(UpdateMe) String pubsubTopic = "projects/{your-project}/topics/{your-topic}";
```

Comment:
Not quite our style, but ok.

Reply:
What's the style for this? Drop the TODO?